WordPressEditor/src/main/res/values/strings.xml

```diff
@@ -9,6 +9,10 @@
     <string name="delete">Delete</string>
     <string name="save">Save</string>
     <string name="discard">Discard</string>
+    <string name="edit">Edit</string>
+    <string name="tap_to_try_again">Tap to try again!</string>
+    <string name="uploading">Uploading…</string>
+    <string name="uploading_gallery_placeholder">[Uploading Gallery]</string>
```

Comment:
Could we keep 'Uploading gallery...' as the message? Also it's probably better to remove the square brackets from the string and add wrap them around the method call in JavaScript instead - that way we can reuse this string in the future if we add visual gallery support and can drop the shortcode in visual mode.